pkgs/applications/version-management/dvc/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, git
+, enableGoogle ? false
+, enableAWS ? false
+, enableAzure ? false
+, enableSSH ? false
+}:
+
+with python3Packages;
+buildPythonApplication rec {
+  pname = "dvc";
+  version = "0.24.0";
+
+  src = fetchFromGitHub {
```

Comment:
Add a comment that there's only a wheel on PyPI.

Reply:
Done.